std/hxcoro/ds/channels/bounded/BoundedChannel.hx

```diff
@@ -0,0 +1,3 @@
+package hxcoro.ds.channels.bounded;
+
+class BoundedChannel<T> extends Channel<T> {}
```

Comment:
Do you foresee that we actually need this class and `UnboundedChannel` in the future for custom functionality? Otherwise I'd find it more natural to just work with `Channel` itself and have its characteristics be determined by its reader and writer configuration.

Reply:
I'm not sure, I think in some earlier iteration these classes did have some stuff in them, but I agree they're odd. Removing them and working with just channel seems sensible.